stone/backends/js_client.py

```diff
@@ -81,11 +87,21 @@ def _generate_route(self, route_schema, namespace, route):
         if route.deprecated:
             self.emit(' * @deprecated')
 
+        return_type = None
+        if self.args.wrap_response_in:
+            if route.result_data_type.__class__ != Void:
```

Comment:
Isn't there value in still doing this for Void type? e.g. if you are providing access to headers with a Response wrapper class like we do in the SDK?